src/uct/api/uct.h

```diff
@@ -797,15 +797,15 @@ void uct_worker_progress_register_safe(uct_worker_h worker, ucs_callback_t func,
  * @ingroup UCT_CONTEXT
  * @brief Remove a slow path callback function from worker's progress.
  *
- * If 'id_p' points to a non-NULL handle, remove a callback which was previously
- * added by @ref uct_worker_progress_register_safe.
+ * If 'id_p' does not point to UCS_CALLBACKQ_ID_NULL, remove a callback which
```

Comment:
More re-wording:
  If *id_p is not equal to UCS_CALLBACKQ_ID_NULL, remove a callback which